lib/ui/window.dart

```diff
@@ -225,6 +225,32 @@ abstract class FlutterView {
   ///   applications.
   WindowPadding get padding => viewConfiguration.padding;
 
+  /// Areas of the display that are obstructed by hardware features. This list is
+  /// populated only on Android. If the device has no display features, this list
+  /// is empty.
+  ///
+  /// The space in which the [DisplayFeature.bounds] are defined includes all screens
+  /// and the space between them. For a dual-screen device, this means that the space
```

Comment:
We should probably not limit to dual screen. Who knows what the future could bring :)